olp-cpp-sdk-core/src/http/curl/NetworkCurl.cpp

```diff
@@ -376,6 +386,9 @@ SendOutcome NetworkCurl::Send(NetworkRequest request,
                               Network::DataCallback data_callback) {
   if (!Initialized()) {
     if (!Initialize()) {
+      OLP_SDK_LOG_ERROR(kLogTag, "Can't send the request with url="
```

Comment:
Is this really a error or rather a warning?

Reply:
As we can't continue the operation, I thought about Error as it used also in other places in NetworkCurl.

Reply:
If it applies to all requests then error, if only to one request then warning.

Reply:
Not sure whether I correctly understood you. In this case, all requests will be failed, cause `Initialize() `most likely will be having some permanent problem.

Reply:
But then you also have error logs in Initialize() which will be called on each Send() until it succeeds. This is a tricky one. Then leave error.